apps/src/StudioApp.js

```diff
@@ -1810,6 +1810,19 @@ StudioApp.prototype.getUnfilledFunctionalBlockError = function (topLevelType) {
   }
 };
 
+/**
+ * @returns {boolean} True if we have a function or variable named "" (empty string)
+ */
+StudioApp.prototype.hasEmptyFunctionOrVariableName = function () {
+  return Blockly.mainBlockSpace.getTopBlocks().some(function (block) {
+    if (block.type !== 'functional_definition') {
+      return false;
+    }
+
+    return block.getProcedureInfo().name === '';
```

Comment:
Good catch. It's possible that we should never actually hit return null, but good to be able to handle it.